tools/peptideshaker/searchgui.xml

```diff
@@ -184,139 +346,534 @@
+                <param name="tide_min_spectrum_mz" type="float" value="0.0"
+                    label="TIDE: Minimum Spectrum m/z" help="Set the minimum spectrum m/z value for a spectrum to be considered"/>
+                <param name="tide_max_spectrum_mz" type="float" value="100000.0"
+                    label="TIDE: Maximum Spectrum m/z" help="Set the maximum spectrum m/z value for a spectrum to be considered"/>
+                <param name="tide_min_spectrum_peaks" type="integer" value="20"
+                    label="TIDE: Minimum Spectrum Peaks" help="Set the minimum amount of peaks in a spectrum for it to be considered"/>
+  	<param name="tide_spectrum_charges" label="TIDE: Spectrum Charges" type="select" help="Select what precursor charges should be taken into account for matching">
+			                	<option value="1" >1</option>
+			                	<option value="2" >2</option>
+			                	<option value="3" >3</option>
+			                	<option value="all" selected="True">all</option>
+	</param>
+                <param name="tide_remove_prec" type="boolean" truevalue="1" falsevalue="0" checked="false"
+                    label="TIDE: Remove Precursor" help="If true, the peak that corresponds to the precursor mass is excluded"/>
+                <param name="tide_remove_prec_tol" type="float" value="1.5"
+                    label="TIDE: Remove Precursor Tolerance" help="Choose the threshold for precursor mass searching (for precursor peak removal)"/>
+                <param name="tide_progress_indicator" type="integer" value="1000"
+                    label="TIDE: Progress Indicator" help="Choose the progress indicator frequency  (in number of fragmentation spectra processed)"/>
+                <param name="tide_use_flanking" type="boolean" truevalue="1" falsevalue="0" checked="false"
+                    label="TIDE: Use Flanking" help="Includes two flanking peaks on either side of each b- and y-ion to compute the XCorr"/>
+                <param name="tide_use_neutral_losses" type="boolean" truevalue="1" falsevalue="0" checked="false"
+                    label="TIDE: Neutral Losses" help="Includes fragment peaks with neutral losses to perform the matching"/>
+                <param name="tide_mz_bin_width" type="float" value="0.02"
+                    label="TIDE: mz Bin Width" help="Choose bin size to analyze the fragmentation spectrum"/>
+                <param name="tide_mz_bin_offset" type="float" value="0.0"
+                    label="TIDE: mz Bin Offset" help="Choose bin offset to analyze the fragmentation spectrum"/>
+                <param name="tide_concat" type="boolean" truevalue="1" falsevalue="0" checked="false"
+                    label="TIDE: Concat Target and Decoy" help="If true, the target results are concatenated with the decoy results"/>
+                <param name="tide_export_text" type="boolean" truevalue="1" falsevalue="0" checked="true"
```

Comment:
Please consider to put these output options into a select box with `multiple="True" options="True"`